lib/daru/vector.rb

```diff
@@ -1548,5 +1576,13 @@ def resort_index vector_index, opts
       end
         .tap { |res| res.reverse! unless opts[:ascending] }
     end
+
+    def table data
+      Formatters::Table.format(
```

Comment:
I used the existing `Daru::Formatters::Table` class to generate the table but had to use it inside a wrapper method with private scope, since using it directly in the `object_summary` was increasing the ABC Metrics.